evil-surround.el

```diff
@@ -157,7 +186,7 @@ See also `evil-surround-inner-overlay'."
         (while (looking-at regexp) (forward-char))
         (evil-set-range-beginning range (point))
         (goto-char (evil-range-end range))
-        (while (looking-back regexp) (backward-char))
+        (while (looking-back regexp nil) (backward-char))
```

Comment:
what is the diffence? Isn't limit nil by default, since is optional?

Reply:
Weird, and i get a byte-compile warning, but the function definition sais that LIMIT is optional.
I use Emacs 27.0.50. 
Don't you get that warning?

Reply:
I use emacs 25.3. this is compile warning is really weird.